### PR TITLE
Added feature detect native lazy loading img

### PR DESCRIPTION
**Pull request by horprogs**
Detects, that image supports tag `loading` (native lazy loading). Returns true if exists property `loading` in `HTMLImageElement.prototype`. Otherwise, returns false.

- Chrome Version 76.0.3805.0 (Official Build) canary (64-bit) with flag #enable-lazy-image-loading (result: true)
- Chrome Version 74.0.3729.169 (Official Build) (64-bit) (result: false)
- Safari 12.1.1 (13607.2.6.1.2) (result: false)

### CODE REVIEW

File: feature-detects/img/loading.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*!
+{
+  "name": "image native lazy loading",
+  "property": "loading",
+  "caniuse": "loading",
```
Comment (rejas) [inline feature-detects/img/loading.js]: the caniuse property "loading-lazy-attr" (https://caniuse.com/#feat=loading-lazy-attr)

File: feature-detects/img/loading.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/*!
+{
+  "name": "image native lazy loading",
```
Comment (rejas) [inline feature-detects/img/loading.js]: better: image and iframe native lazy loading

File: feature-detects/img/loading.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "property": "loading",
```
Comment (rejas) [inline feature-detects/img/loading.js]: I'd like to use "lazyloading" for this (which you also use in the actual test down below)

File: feature-detects/img/loading.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "caniuse": "loading",
+  "tags": ["image", "lazy", "loading"],
+  "notes": [{
+    "name": "Native image lazy-loading for the web",
+    "href": "https://addyosmani.com/blog/lazy-loading/"
+  }]
+}
+!*/
+/* DOC
+Test for the loading attribute of images
```
Comment (rejas) [inline feature-detects/img/loading.js]: ... and iframes

**Comment by horprogs** [comment]
Thx you, I fixed it

File: lib/config-all.json [hunk 5]
```diff
@@ -204,6 +204,7 @@
     "img/crossorigin",
     "img/jpeg2000",
     "img/jpegxr",
+    "img/loading",
```
Comment (rejas) [inline lib/config-all.json]: just one more thing: rename the file from just "loading" to "lazyloading" Makes it clearer what the feature does and correpsonds to the property too.

Comment (horprogs) [reply lib/config-all.json]: No problem. Renamed.

**Comment by rejas** [comment]
Thx a lot!

**Pull request merged**